Tests/BoundariesTests/TestReducer.swift

```diff
@@ -0,0 +1,129 @@
+import Boundaries
+import Dispatch
+import Optics
+import Prelude
+
+struct CounterState {
+  var count: Int
+}
+
+enum CounterAction {
+  case incr
+  case decr
+}
+
+let counterReducer = Reducer<CounterState, CounterAction, MyEffect> { action, state in
+  switch action {
+  case .incr:
+    return (
+      state |> \.count +~ 1,
+      .parallel(
+        [
+          .execute(.print("incr")),
```

Comment:
Wondering if we can leverage `EffectProtocol` to avoid the need to wrap things in `.execute` so often.

Reply:
yeah that'd be nice